ios/CodePush/CodePushDownloadHandler.m

```diff
@@ -6,12 +6,14 @@ @implementation CodePushDownloadHandler {
 }
 
 - (id)init:(NSString *)downloadFilePath
+usingQueue:(dispatch_queue_t)usingQueue
 progressCallback:(void (^)(long long, long long))progressCallback
 doneCallback:(void (^)(BOOL))doneCallback
 failCallback:(void (^)(NSError *err))failCallback {
     self.outputFileStream = [NSOutputStream outputStreamToFileAtPath:downloadFilePath
                                                               append:NO];
     self.receivedContentLength = 0;
```

Comment:
This isn't related to this PR, but I feel like we could convert all of these properties into ivars and it would clean up the header a little bit.